JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -1710,14 +1710,14 @@ class AppInsightsTests extends TestClass {
         });
 
         this.testCase({
-            name: "Ajax - root/parent id are set and passed correctly",
+            name: "Ajax - Request-Id is set and passed correctly",
             test: () => {
                 var snippet = this.getAppInsightsSnippet();
                 snippet.disableAjaxTracking = false;
                 snippet.disableCorrelationHeaders = false;
                 snippet.maxBatchInterval = 0;
                 var appInsights = new Microsoft.ApplicationInsights.AppInsights(snippet);
-                var trackStub = this.sandbox.spy(appInsights, "trackDependency");
```

Comment:
do we have any tests for `trackDependency` or only for `trackDependencyData`?

Reply:
i left unit tests for trackDependency without any changes as they're testing public API, but unit tests for ajax collection now for trackDependencyData - so both are covered now. i can make a duplicate of trackDependency unit tests for trackDependencyData as well.